src/webapp/logic/sheetBuilder.ts

```diff
@@ -341,10 +341,12 @@ export class SheetBuilder {
                     }
 
                     if (description !== undefined) {
-                        sheet.cell(itemRow, columnId).comment(description, {
-                            height: "100pt",
-                            width: "160pt",
-                        });
+                        // TODO: adding description as a comment is generating an error in excel
+                        // More details here: https://github.com/EyeSeeTea/Bulk-Load/pull/348
+                        // sheet.cell(itemRow, columnId).comment(description, {
+                        //     height: "100pt",
+                        //     width: "160pt",
+                        // });
```

Comment:
When the time arrives to debug the problem:

https://github.com/EyeSeeTea/xlsx-populate/commit/1b536db193866700a795215424a384ea1aa3359e

And for debugging, check the scripts here: https://github.com/EyeSeeTea/xlsx-populate/tree/master/scripts

Particularly, extract-xlsx.sh and create-xlsx.sh, you can test easily changes in the XML structure and see if LO/Excel open the file without errors. With diff-xlsx.sh you can diff 2 xlsx, for example one can be the original and the second one the fixed by Excel after a Save As (if that was possible)